example/flaskr/ingredients.sql

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I think you might have inadvertently changed the name of this file in your branch when you pulled it down as this file already exists in the repo as `/flaskr/ingredient.sql` (without the 's').
I think we should be good to remove this.

Reply:
I don't know how that happened. I never went and edited file names. Weird, but fixed